mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
I thought we're using csc.exe from 4.5 anyway already, that's why I asked in https://github.com/mono/mono/pull/5251#discussion_r129285608 since using the AOTd files is important.

Which makes me wonder why we need this at all :)